### ISSUE TITLE
Windows Defender Issue

### ISSUE THREAD

**Issue by Cato917** [34736384/genshin-fps-unlock#304]
Any updates on getting it to stop triggering WD notifs?
I've whitelisted it multiple times and WD refuses to allow it to run more than once without shitting the bed and deleting it even if it's an exluded file, getting annoying just redownloading it over and over since i've got to be quick enough to bypass WD when running it before it deletes it from my downloads folder, they get automatically quarantined and no i'm not turning off WD and i'm also not playing a mobile game at 60fps either!!!
tldr: can't even whitelist this shit because WD just bypasses every whitelist/exlusion for some reason and instantly deletes it after the first run.

**Comment by 34736384** [comment]
soon, in one or two days
rewriting it in net core, VT still spits out two detections tho

**Comment by sss-ryun** [comment]
Build it yourself. WD has been silent since I did

**Comment by 34736384** [comment]
updated
